### PR TITLE
Flag secure in all activities that extends BaseActionBarActivity

### PR DESCRIPTION
**Pull request by jodson**

### CODE REVIEW

File: src/org/thoughtcrime/securesms/BaseActionBarActivity.java [hunk 1]
```diff
@@ -42,6 +51,17 @@ public boolean onKeyUp(int keyCode, @NonNull KeyEvent event) {
     return super.onKeyUp(keyCode, event);
   }
 
+  //If some class need to not use flag_secure for some reason. It's just override and do nothing.
+  protected void initializeScreenshotSecurity() {
```
Comment (mcginty) [inline src/org/thoughtcrime/securesms/BaseActionBarActivity.java]: make it private and remove the comment - we can change to protected if the need arises

Comment (jodson) [inline src/org/thoughtcrime/securesms/BaseActionBarActivity.java]: ok

**Comment by jodson** [comment]
Done

**Comment by jodson** [comment]
@mcginty any answer?

File: src/org/thoughtcrime/securesms/BaseActionBarActivity.java [hunk 2]
```diff
@@ -42,6 +51,16 @@ public boolean onKeyUp(int keyCode, @NonNull KeyEvent event) {
     return super.onKeyUp(keyCode, event);
   }
 
+  private void initializeScreenshotSecurity() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH &&
+            TextSecurePreferences.isScreenSecurityEnabled(this))
```
Comment (mcginty) [inline src/org/thoughtcrime/securesms/BaseActionBarActivity.java]: align this line with the `(`

**Comment by moxie0** [comment]
sorry for missing the earlier discussion, what is it that you're trying to accomplish with this change?

**Comment by dcherian** [comment]
This one? https://github.com/WhisperSystems/Signal-Android/issues/3327

**Pull request closed without merging**